chain/network-primitives/src/types.rs

```diff
@@ -584,15 +585,17 @@ impl NetworkConfig {
         }
     }
 
-    pub fn verify(&self) {
+    pub fn verify(&self) -> Result<()> {
```

Comment:
can you start a binary with mainnet/testnet config to see that these things don't crash ?

(or can you check the logs that none of these errors are printed?)

Reply:
@mm-near Yes, I started testnet.